examples/pln/johnny/Assertions.scm

```diff
@@ -0,0 +1,50 @@
+(ConceptNode "Johnny@444") (stv 0.99 1)
+
+(ConceptNode "Sweets@555") (stv 0.99 1)
+
+(ConceptNode "day@222") (stv 0.99 1)
+
+(ConceptNode "he@345") (stv 0.99 1)
+
+(ConceptNode "himself@456") (stv 0.99 1)
+
+(ConceptNode "risk@44") (stv 0.99 1)
+
+(ConceptNode "diabetes@66") (stv 0.99 1)
+
+(ImplicationLink (stv 0.99 1)
+	(EvaluationLink
+		(PredicateNode "every@111")
+        (ListLink
+			(ConceptNode "day@222")
+			(EvaluationLink
+				(PredicateNode "eating@333")
+				(ListLink
+					(ConceptNode "Johnny@444")
+					(ConceptNode "Sweets@555")))))
+	(EvaluationLink
+		(PredicateNode "at@123")
+		(ListLink
+			(EvaluationLink
+				(PredicateNode "placing@234")
+				(ListLink
+					(ConceptNode "he@345")
+					(ConceptNode "himself@456")))
+			(ConceptNode "risk@44"))))
+
+(EvaluationLink (stv 0.99 1)
+	(PredicateNode "for@55")
+	(ListLink
+		(ConceptNode "risk@44")
+		(ConceptNode "diabetes@66")))
+
+(NotLink (stv 0.99 1)
+	(EvaluationLink
+		(PredicateNode "every@111")
+        (ListLink
+			(ConceptNode "day@222")
+			(EvaluationLink
+				(PredicateNode "eating@333")
+				(ListLink
+					(ConceptNode "Johnny@444")
+					(ConceptNode "Sweets@555"))))))
```

Comment:
Lol.  There's so much that's right and so much that's wrong about those!
This is why there were quite a few of the old
R2L rules that I didn't even bother porting at all.  Like the not-rule that
was there.  I mean, what?  I will have to look at
both the LG and relex output also to figure out exactly what it's doing,
here.  I mean it looks to me, like the LG or relex output
for those sentences is probably not what it should be to interface
correctly with the R2L rules . . .  It's like trying
to put together one of those puzzles made of wooden blocks, where as soon
as you fit one piece in, three of the ones
that were in before, fall out . . .

On Wed, May 27, 2015 at 6:31 PM, Amen Belayneh <notifications@github.com>
wrote:

> In examples/pln/johnny/Assertions.scm
> <https://github.com/opencog/opencog/pull/1594#discussion_r31120615>:
>
> > +(EvaluationLink (stv 0.99 1)
> > +	(PredicateNode "for@55")
> > +	(ListLink
> > +		(ConceptNode "risk@44")
> > +		(ConceptNode "diabetes@66")))
> > +
> > +(NotLink (stv 0.99 1)
> > +	(EvaluationLink
> > +		(PredicateNode "every@111")
> > +        (ListLink
> > +			(ConceptNode "day@222")
> > +			(EvaluationLink
> > +				(PredicateNode "eating@333")
> > +				(ListLink
> > +					(ConceptNode "Johnny@444")
> > +					(ConceptNode "Sweets@555"))))))
>
> @anitzkin <https://github.com/anitzkin> as requested, the r2l outputs for
> the two sentences
> https://gist.github.com/AmeBel/1ed86e75240013cb9f4f
> https://gist.github.com/AmeBel/f5bd8c73f5ec7dd60370
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/1594/files#r31120615>.
>